### DISCUSSION TITLE
Custom logo and meta

### DISCUSSION THREAD

**Discussion by danilopolani** [nuxt-content/docus#880] [Q&A]
I'm creating a new docs project with Docus but I have a few questions and I can't find the answer anywhere.

1. How we can customize the logo? Maybe will be fixed by #874 , but still can't understand from that MR how we can set the logo image or even a custom component, since the `header.logo` property is, right now, a boolean.
2. Can we add in any way more meta to the head of all pages? E.g. an analytics tracking snippet.
3. What's the best way to write CSS? For example if I want to change the background only of the "homepage".

Thank you :)

**Comment by MuhammadM1998** [comment]
Maybe I can help on the first point.
 
Enable header logo in `app.config` then add a `Logo.vue` component file in `components/global` as following

***app.config.ts***
```javascript
export default defineAppConfig({
  docus: {
    header: { logo: true },
    // rest of the config
  } 
});
```

***components/global/Logo.vue:***
```vue
<template>
  <img src="/your-logo.png" /> // your-logo.png should be in public folder
</template>
```

I'm using this in my docs and working as expected, there's a [docs section](https://docus.dev/introduction/configuration#customizing-the-logo) too on this regard
